<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>981</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="coverImgLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>881</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="viewAlbumsPage">
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Genre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Year of creation:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teSearchGenre">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teSearchYear">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teSearchAuthor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="albumList">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>581</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="newAlbumPageButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>430</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add a new album</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goToDetailButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>500</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to detail of selected album</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteAlbumButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>500</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete the selected album</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetSearchButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addAlbumPage">
     <widget class="QPushButton" name="addAlbumButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add the album to the database</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Year of creation:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teAlbumName">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teGenre">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Genre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>List of songs:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teListOfSongs">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teAuthorName">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="selectImageButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Select cover image</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="teYearOfCreation">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Author name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Album name:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="albumLibPageButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>420</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Abum library</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="detailAlbumPage">
     <widget class="QLabel" name="albumName">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="albumYear">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="albumGenre">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="albumAuthor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="albumSongList">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>311</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="fromDetailToLibButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Album library</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Songs:</string>
      </property>
     </widget>
     <widget class="QLabel" name="albumImage">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate_Album">
    <property name="title">
     <string>Create Album</string>
    </property>
   </widget>
   <addaction name="menuCreate_Album"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
